src/utils/daos_hdlr.h

```diff
@@ -62,6 +66,25 @@ enum obj_op {
 	OBJ_DUMP
 };
 
+#define NUM_DIRENTS 24
+#define MAX_FILENAME 256
+
+struct dfs_daos_t {
+	dfs_obj_t* dir;
```

Comment:
I'm getting this compile error: (master seems to compile without issue)
```
In file included from src/utils/daos_autotest.c:38:0:
src/utils/daos_hdlr.h:73:2: error: unknown type name ‘dfs_obj_t’
  dfs_obj_t* dir;
```

Reply:
> I'm getting this compile error: (master seems to compile without issue)
> 
> ```
> In file included from src/utils/daos_autotest.c:38:0:
> src/utils/daos_hdlr.h:73:2: error: unknown type name ‘dfs_obj_t’
>   dfs_obj_t* dir;
> ```

@daltonbohning honestly not sure what happened there. I was just building this about a week ago, that type is defined in daos_fs.h, which has been added. I'm looking into it now.

Reply:
@daltonbohning are you adding daos_fs.h?

Reply:
It seems that `daos_hdlr.h` is now included from `daos_autotest.c`. And `daos_hdlr.h` doesn't include `daos_fs.h`, so the types aren't defined.
I wonder if we can actually just put the structs from `daos_hdlr.h` into `daos_hdlr.c`, since they are only being used within `daos_hdlr.c` anyway?